extensions/git/src/commands.ts

```diff
@@ -3665,11 +3660,9 @@ export class CommandCenter {
 					case GitErrorCodes.PushRejected:
 						message = l10n.t('Can\'t push refs to remote. Try running "Pull" first to integrate your changes.');
 						break;
-					case GitErrorCodes.ForcePushRejected:
-						message = l10n.t('Can\'t force push refs to remote. Check if force push is not prohibited by remote.');
-						break;
-					case GitErrorCodes.SaferForcePushRejected:
-						message = l10n.t('Can\'t force push refs to remote. Check if force push is not prohibited by the remote or pull the latest changes from the remote branch first.');
+					case GitErrorCodes.ForcePushWithLeaseRejected:
+					case GitErrorCodes.ForcePushWithLeaseIfIncludesRejected:
+						message = l10n.t('Can\'t force push refs to remote. The tip of the remote-tracking branch has been updated since the last checkout. Try running "Pull" first to pull the latest changes from the remote branch first.');
```

Comment:
@lszomoru I believe the pull operation make the matter worse in the case the user intentionally choose the force push instead of the usual one.
How about changing "`Try running "Pull" first to pull the latest changes from the remote branch first.`" to "`Reflect the latest changes from the latest changes from branch first e.g. by trying running "Pull" first.`"?

Reply:
> I believe the pull operation make the matter worse in the case the user intentionally choose the force push instead of the usual one.

I think that the "theme" of these changes was to make force push safer and avoid data loss. In light of that, I think that the suggestion to Pull makes sense. Users that really want to force push will have to either change their settings or use the terminal. 

I am not sure I understand what you mean by "Reflect the latest changes from the latest change". I would suggest that we go ahead with the current message in the PR and then we can adjust it based on user feedback.